File: T8. DATA TYPES AND VARIABLES - EXERCISES (Java)/src/P16_ComparingFloats.java
import java.util.Scanner;

public class P16_ComparingFloats {
    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);

        double num1 = Double.parseDouble(in.nextLine());
        double num2 = Double.parseDouble(in.nextLine());
        final double eps = 0.000001;

        double difference = Math.abs(num1 - num2);
        boolean result = difference < eps;

        if (result) {
            System.out.println("True");
        } else {
            System.out.println("False");
        }
    }
}

File: T8. DATA TYPES AND VARIABLES - EXERCISES/P16.ComparingFloats.cs
using System;

namespace P16.ComparingFloats
{
    class Program
    {
        const double eps = 0.000001;
        static void Main(string[] args)
        {
            double num1 = double.Parse(Console.ReadLine());
            double num2 = double.Parse(Console.ReadLine());
            double difference = Math.Abs(num1 - num2);

            bool result = difference < eps;
            Console.WriteLine(result);
        }
    }
}
